internal/infra/database/database.go

```
@@ -5,59 +5,60 @@ import (
 
-	// Open connection
-	sqldb, err := sql.Open(cfg.Driver, dsn)
+func NewMySQLClient(cfg *Config) (*upbun.DB, error) {
+	sqldb, err := sql.Open("mysql", GetMySQLDSN(cfg))
 	if err != nil {
-		return nil, fmt.Errorf("failed to open database connection: %w", err)
+		return nil, fmt.Errorf("failed to open mysql connection: %w", err)
 	}
-
-	// Test connection
 	if err := sqldb.Ping(); err != nil {
```

golangci-lint (2.11.4)

[error] (*database/sql.DB).Ping must not be called. use (*database/sql.DB).PingContext

(noctx)